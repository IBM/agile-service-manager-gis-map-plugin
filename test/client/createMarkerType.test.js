// Import the module under test
import {createMarkerType} from '../../src/scripts/client/createMarkerType';

describe('createMarkerType', function() {
    test('create', function() {        
        expect(createMarkerType('myType')).toMatchSnapshot();
    });
});