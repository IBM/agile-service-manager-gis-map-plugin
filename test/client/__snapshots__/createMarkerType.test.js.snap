// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`createMarkerType create 1`] = `
Object {
  "affectedRadiusMarkers": Object {},
  "affectedRadiusMarkersLayer": NewClass {
    "_initHooksCalled": true,
    "_layers": Object {},
    "options": Object {},
  },
  "clusterGroup": NewClass {
    "_initHooksCalled": true,
    "_layers": Object {},
    "options": Object {},
  },
  "locations": Array [],
  "locationsMap": Object {},
  "markerType": "myType",
  "markers": Object {},
}
`;
